packages/math-input/src/components/keypad/operators-page/basic-relations-buttons.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import Keys from "../../../data/key-configs";
+import {PlaceHolderButtons, SecondaryKeypadButton} from "../keypad-page-items";
+
+type Props = {
+    onClickKey: (keyConfig: string) => void;
+    placeholder?: boolean;
+};
+
+export const BasicRelations = ({
+    onClickKey,
+    placeholder,
+}: Props): React.ReactElement =>
+    placeholder ? (
+        <PlaceHolderButtons count={3} />
+    ) : (
```

Comment:
Not sure this should be in the BasicRelations component. Wouldn't it make sense to have the parent conditionally render either the PlaceHolderButtons or BasicRelations? I'm also wondering if we need this broken out into a component at all.

Reply:
I had that originally and Jeremy had the opposite note 😅

Reply:
For the PreAlgebra buttons. His justification was if the number of buttons needs in the button set, we might not know to change the placeholder in the parent.